09_Time_Series/Getting_Financial_Data/Exercises_with_solutions_and_code.ipynb

```diff
@@ -85,7 +85,7 @@
    },
    "outputs": [],
    "source": [
-    "stocks = ['AAPL', 'TSLA', 'IBM', 'LNKD']"
+    "stocks = ['AAPL', 'TSLA', 'IBM', 'TWTR', 'LNKD']"
```

Comment:
I couldn't find TWTR although it was menthioned.
(I think you should change other files too.)